Exercise_A/index.html

```diff
@@ -16,7 +16,7 @@
   </head>
   <body>
     <!-- CODE HERE -->
-
+    <a href="about.html"><img src="https://www.kafkabrigade.org.uk/wp-content/uploads/2011/07/button-pic.jpg" alt=""></a>
     <script src="" async defer></script>
   </body>
 </html>
```

Comment:
Well done on linking the image to another HTML document. I love the image that you used!